css/main.css

```diff
@@ -0,0 +1,31 @@
+@charset "UTF-8";
+
+@import url(https://fonts.googleapis.com/css?family=Titillium+Web:700,600);
+@import url(http://lab.clementparis.fr/octicons/octicons.css);
+
+html{
+  height: 100%;
+  box-sizing:border-box;
+}
+
+*, *::after, *::before {
+  box-sizing: inherit;
+
+body {
```

Comment:
I'm not sure if this body or nav styling got brought into the final version of not. Also the html settings did not get brought over. We may need to update this current version.